src/main/java/sic/controller/ProductoController.java

```diff
@@ -139,6 +143,18 @@ public void getListaDePrecios(
     }
   }
 
+  @GetMapping("/productos/reporte")
+  public ResponseEntity<byte[]> getListaProductosSeleccionados(
```

Comment:
The response should be sent by email

Reply:
The use case doesn't require the response to be send by email, the requirement is obtain a report of the chosen products the same way that others like Factura and Pedidos do, the email one it's beacause usualy whitout a product selection the list is to big and the request take to long

Reply:
What if the user selects X (a very big number) products and sends the request? It's gonna crush. We need to cover all possible scenarios

Reply:
Happens that the access to the business email account it's only to very few people, and the new feature it's to vendedor use as "Presupuesto". Now I notice a problem, to use this as a vendedor we have to let it access to the product administration list, then the real solution it's to have a real "Presupuesto" report, this do not solve the original problem and carry a new one.